src/transactionsFactories/tokenManagementTransactionsFactory.ts

```diff
@@ -299,7 +299,7 @@ export class TokenManagementTransactionsFactory {
         tokenIdentifier: string;
         addRoleLocalMint: boolean;
         addRoleLocalBurn: boolean;
-        addRoleESDTTransferRole?: boolean;
+        addRoleESDTTransferRole: boolean;
```

Comment:
Indeed. It's a change that slightly breaks the current public API, but **it's a fixing change**. Not passing the `addRoleESDTTransferRole` parameter is worse than being forced to (when upgrading to the new minor version of the library).

We should document this in the release notes, though.